web_src/js/features/repo-settings.ts

```diff
@@ -154,4 +155,5 @@ export function initRepoSettings() {
   initRepoSettingsCollaboration();
   initRepoSettingsSearchTeamBox();
   initRepoSettingsGitHook();
+  initRepoBranchesSettings();
```

Comment:
Other function names are unified to `initRepoSettingsXxx` 🤣 

anyway, just a nit, no block, feel free to update or just merge (already 2 approvals)

Reply:
... well there will be a refactor that catches it at some point